app/qml/FeatureForm.qml

```diff
@@ -407,9 +407,12 @@ Item {
 
     Item {
       id: fieldContainer
-      visible: Type === FormItemType.Field
+
+      property bool shouldBeVisible: Type === FormItemType.Field || Type === FormItemType.Relation
```

Comment:
it is definitely ok to do it this way, but just for consideration to have 2 separate items , one for fields (visible: Type === FormItemType.Field) and one for relations  (visible: Type === FormItemType.Relation), since not sure how much implementation/qml they share?

also it would be good to finally use components and try to detangle/un-spagetty the feature form to some components in separate files

Reply:
That is a good point. They share field name and some properties from form model like featurepair or config. However, most of the other properties are not used. Even checkboxes for _Reuse last value_ should not be there. 

Yes, feature form needs to be cleaned. Let's have a discussion after relations. My plan is to at least consolidate the styling within relations.